packages/compass-editor/src/ace/query-autocompleter.test.ts

```diff
@@ -1,273 +1,158 @@
-import type { Ace } from 'ace-builds';
```

Comment:
Both these tests and stage-autocompleter tests were not changed, they are still testing the same things as before, I just moved them to use this `setupCompleter` test helper I added for aggregation-autocompleter tests